sys/fuchsia/mtrace.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <zircon/syscalls.h>
+
+zx_mtrace_control(handle zx_handle, kind int32, action int32, options int32, ptr ptr[in, array[int8]], ptr_size len[ptr])
```

Comment:
For this syscall and lots of other syscalls in the new PRs, is this zx_handle can be of absolutely any type, or some specific type is required? If a special type is required, then it makes sense to specify the exact subtype of handle.

Reply:
Fix coming up in a moment.